spec/breadcrumbs/validator_spec.rb

```diff
@@ -0,0 +1,254 @@
+# encoding: utf-8
+require 'spec_helper'
+
+require 'bugsnag/breadcrumbs/types'
+require 'bugsnag/breadcrumbs/validator'
+
+RSpec.describe Bugsnag::Breadcrumbs::Validator do
+  before do
+    @automatic_breadcrumb_types = [
```

Comment:
I prefer using `let` ( http://www.betterspecs.org/#let ) for these instead of instance variables, as it tends to make it easier to refactor tests in case local variables are more appropriate, and to override values in certain situations.